beam-sqlite/beam-sqlite.cabal

```diff
@@ -1,5 +1,5 @@
 name:                beam-sqlite
-version:             0.5.0.0
+version:             0.6.0.0
```

Comment:
Why do we need to bump the major version number for this?

Reply:
Habit after a new release. I suppose the changes only mandate 0.5.1.0

Reply:
`Database.Beam.Postgres.Connection` exports `PgF` though so I think it is backward breaking...

Reply:
Oh but `Database.Beam.Postgres.Connection` is hidden and it doesn't look like `PgF(..)` gets re-exported anywhere. It would help if these module names made hiddenness a bit clearer.